command/analyzer/run/processors/silencers.go

```diff
@@ -0,0 +1,106 @@
+package processors
+
+import (
+	"regexp"
+	"strings"
+)
+
+var languagesMeta map[string]LanguageMeta
+
+// for the extension e.g .py, .go, .js etc build a map of compiled
+// regex with the corresponding comment identifier in regex.
+var regexMap map[string]regexp.Regexp
+
+// Silencers data structure
+type IssueSilencer struct {
```

Comment:
This isn't needed. We only need `skipcq` to be supported, and only for the issues raised by this analyzer.

Also,
an analyzer issue code in the analyzer is only "I0001". But the DeepSource issue will be `ABC-I0001`, where `ABC` can be any string, with which we'll uniquely identify the analyzer. When we run the skipcq processor, we only look for the tag `I0001` in the code, in case `I0001` is raised by the analyzer. However, the skipcq tag may also look like `skipcq: ABC-I0001`. We'll have to skip these issues as well.

Reply:
Taking a note to implement this internally.